sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -354,15 +364,6 @@ export interface TableCreateHeaders {
     version?: string;
 }
 
-// @public
-export interface TableCreateHeaders {
```

Comment:
This is not a deletion. The TableCreateHeaders was defined twice earlier. Now, one of it is removed.